core/src/main/scala/io/aiven/guardian/kafka/KafkaClient.scala

```diff
@@ -43,7 +44,11 @@ class KafkaClient(
 
   private[kafka] val consumerSettings = {
     val base = ConsumerSettings(system, new ByteArrayDeserializer, new ByteArrayDeserializer)
-    configure.fold(base)(block => block(base))
+    configure
+      .fold(base)(block => block(base))
+      .withProperties(
+        ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG -> false.toString
```

Comment:
We want to have full control of committing the Kafka cursors, if this isn't set to false than Kafka will periodically do commits in backround (outside of our control) in addition to our manual commits after an S3 chunk is uploaded